tests/e2e/specs/settings.spec.js

```diff
@@ -0,0 +1,114 @@
+ */
+const { test, expect } = require( '@playwright/test' );
+
+/**
+ * Internal dependencies
+ */
+import SettingPage from '../utils/pages/settings.js';
+import ElementLocators from '../utils/element-locators.js';
+
+/**
+ * @type {import('../utils/pages/settings.js').default} settingPage
+ */
+let settingPage = null;
+
+/**
+ * @type {import('../utils/element-locators.js').default} onboardingPage
+ */
+let locator = null;
+
+/**
+ * @type {import('@playwright/test').Page} page
+ */
+let page = null;
+
+test.describe( 'Snapchat Settings', () => {
+	test.use( { storageState: process.env.ADMINSTATE } );
+
+	test.beforeAll( async ( { browser } ) => {
+		page = await browser.newPage();
+		settingPage = new SettingPage( page );
+		locator = new ElementLocators( page );
+
+		await settingPage.mockJetpackConnected();
+		await settingPage.mockSnapchatConnection( { status: 'connected' } );
+		await settingPage.mockOnboardingSetup( {
+			status: 'connected',
+			step: 'accounts',
+		} );
+	} );
+
+	test.afterAll( async () => {
+		await settingPage.closePage();
+	} );
+
+	test( 'Shows all sections', async () => {
+		settingPage.goto();
+
+		await expect(
+			page.locator( '.sfw-section__header', {
+				hasText: 'Product Catalog',
+			} )
+		).toBeVisible();
+		await expect(
+			page.locator( '.sfw-section__header', {
+				hasText: 'Track Conversions',
+			} )
+		).toBeVisible();
+		await expect(
+			page.locator( '.sfw-section__header', {
+				hasText: 'Manage Snapchat Connection',
+			} )
+		).toBeVisible();
+	} );
+
+	test( 'Toggle conversion tracking', async () => {
+		settingPage.goto();
+
+		await locator.getCapiCheckbox().click();
+		await expect( locator.getCapiCheckbox() ).toBeEnabled();
+		await expect(
+			page
+				.getByText(
+					'Enhanced Conversions status updated successfully.'
+				)
+				.first()
+		).toBeVisible();
+
+		await locator.getCapiCheckbox().click();
+		await expect( locator.getCapiCheckbox() ).toBeDisabled();
+		await expect(
+			page
+				.getByText(
+					'Enhanced Conversions status updated successfully.'
+				)
+				.first()
+		).toBeVisible();
+	} );
+
+	test( 'Snapchat card details', async () => {
+		const payload = {
+			org_id: '244753a0-2021-482c-af9b-dd6e7677d562',
+			org_name: 'SnapForWooV105',
+			ad_acc_id: '89b3e14b-bac9-409e-857c-ab006cd1c96e',
+			ad_acc_name: 'SnapForWooV105 Self Service',
+			pixel_id: 'fd014a21-2e25-41a8-9e12-de8c9fe512b4',
+		};
+
+		await settingPage.mockSnapchatAccount( payload );
+		settingPage.goto();
+
+		await expect( locator.getSnapchatAccountCard() ).toContainText(
+			'Organization: SnapForWooV105'
+		);
+		await expect( locator.getSnapchatAccountCard() ).toContainText(
+			'Ads Account: SnapForWooV105 Self Service (89b3e14b-bac9-409e-857c-ab006cd1c96e)'
+		);
+		await expect( locator.getSnapchatAccountCard() ).toContainText(
+			'Pixel ID: fd014a21-2e25-41a8-9e12-de8c9fe512b4'
+		);
+		await expect( locator.getSnapchatConnectedLabel() ).toBeVisible();
+	} );
```

Comment:
It would be nice to add an additional test case for disconnecting the Snap account from this UI here.

Reply:
Addressed in [52a9db7](https://github.com/woocommerce/snapchat-for-woocommerce/pull/13/commits/52a9db72a079c74f68a17161d63f4953437e319f)